jdi-light/src/main/java/com/epam/jdi/light/settings/DriverSettings.java

```diff
@@ -1,45 +1,58 @@
 package com.epam.jdi.light.settings;
 
-import com.epam.jdi.light.driver.WebDriverFactory;
 import com.epam.jdi.light.driver.get.*;
 import com.epam.jdi.tools.func.JFunc1;
 import com.epam.jdi.tools.map.MapArray;
 import org.openqa.selenium.PageLoadStrategy;
 import org.openqa.selenium.WebDriver;
 
+import static com.epam.jdi.light.driver.WebDriverFactory.getWebDriverFactory;
 import static com.epam.jdi.light.driver.get.DriverInfos.*;
 import static com.epam.jdi.light.driver.get.DriverTypes.*;
-import static com.epam.jdi.light.driver.get.DriverVersion.*;
-import static com.epam.jdi.light.driver.get.GetDriverTypes.*;
-import static com.epam.jdi.light.driver.get.Platform.*;
-import static com.epam.jdi.light.settings.JDISettings.*;
-import static com.epam.jdi.tools.PathUtils.*;
-import static com.epam.jdi.tools.map.MapArray.*;
-import static com.epam.jdi.tools.pairs.Pair.*;
-import static org.openqa.selenium.PageLoadStrategy.*;
+import static com.epam.jdi.light.driver.get.DriverVersion.LATEST;
+import static com.epam.jdi.light.driver.get.GetDriverTypes.DOWNLOAD;
+import static com.epam.jdi.light.driver.get.Platform.X64;
+import static com.epam.jdi.light.settings.CommonSettings.getCommonSettings;
+import static com.epam.jdi.tools.PathUtils.mergePath;
+import static com.epam.jdi.tools.map.MapArray.map;
+import static com.epam.jdi.tools.pairs.Pair.$;
+import static org.openqa.selenium.PageLoadStrategy.NORMAL;
 
 public class DriverSettings {
-    public String name = CHROME.name;
-    public String version = LATEST.value;
+    public DriverSettings() {
+        name = CHROME.name;
+        version = LATEST.value;
+        platform = X64;
+        getType = DOWNLOAD;
+        path = "C:/Selenium";
+        pageLoadStrategy = NORMAL;
+        screenSize = new ScreenSize();
+        downloadsFolder = mergePath(getCommonSettings().testPath, "resources", "downloads");
+        types = map(
+                $(CHROME.name, CHROME_INFO),
+                $(FIREFOX.name, FIREFOX_INFO),
+                $(IE.name, IE_INFO),
+                $(EDGE.name, EDGE_INFO),
+                $(OPERA.name, OPERA_INFO),
+                $(SAFARI.name, SAFARI_INFO)
+        );
+        capabilities = new CapabilitiesSettings();
+    }
+
+    public String name;
+    public String version;
     public OsTypes os;
-    public Platform platform = X64;
-    public GetDriverTypes getType = DOWNLOAD;
-    public String path = "C:/Selenium";
-    public PageLoadStrategy pageLoadStrategy = NORMAL;
-    public ScreenSize screenSize = new ScreenSize();
+    public Platform platform;
+    public GetDriverTypes getType;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5482899)